exporter/lokiexporter/README.md

```diff
@@ -17,12 +17,19 @@ Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/).
 The following settings are required:
 
 - `endpoint` (no default): The target URL to send Loki log streams to (e.g.: `http://loki:3100/loki/api/v1/push`).
+- `default_labels_enabled` (optional): The map that allows to disable default labels: `exporter`, `job`, `instance`, `level`. 
+If `default_labels_enabled` is omitted then default labels will be added. If one of the labels is omitted in `default_labels_enabled` then this label will be added.
+**Important to remember**: 
+If all default labels are disabled and there are no other labels added then the log entry would be dropped because at least one label should be present to successfully put the log record into Loki 
 
 Example:
 ```yaml
 exporters:
   loki:
     endpoint: https://loki.example.com:3100/loki/api/v1/push
+    default_labels_enabled:
```

Comment:
Perhaps we could have it differently? Have a set of default labels, and let it be overridden by users? Like:

```
default_labels:
  exporter: OTLP
```

This would cause `exporter=OTLP` to be the only label to be included.

Reply:
cc @gillg

Reply:
Yeah ! Good to see it :)

I agree with the fact we can cover more cases with a string instead of a book flag. So "what can the more can the less" as I would say in French.
The only remaining question is for backward compatibility, I remember you was not fan to change the actual default label, so if this new config is absent we just assume there is still the hardcoded label otlp ? So how to remove it completely by config ?

Reply:
there is a possibility to override exporter label with attributes processor:
```
processor:
  attributes:
     actions:
      - action: insert
         key: loki.attribute.labels
         value: exporter
      - action: insert
         key: exporter
         value: overridden
```
I would prefer to keep default_labels_enabled as a map[string]bool to allow removing default labels. To override label values we can use attributes/resource processors

Reply:
Makes sense and make it consistent at least. 👍 
So to override you need to disable the default and inject the same name by processors (or if one of the default is in `loki.attribute.labels` you automaticaly override the default ?).

Reply:
> if one of the default is in loki.attribute.labels you automaticaly override the default ?)

yes, that's correct